style.css

```diff
@@ -5,9 +5,21 @@ h1 {
 h3 {
 	text-transform: uppercase;
 }
-li {
-	overflow: hidden;
-	padding: 20px 0;
-	border-bottom: 1px solid #eee;
+ul {
+	margin: 0;
+	padding: 0;	
 }
-
+ul li {
```

Comment:
**Just for clarification**

Make the list items unselectable for a cleaner UX. Use `user-select: none`.